examples/website-demos/social-messaging/package.json

```diff
@@ -13,6 +13,8 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
+    "stream-chat": "^3.11.0",
+    "stream-chat-react": "^6.4.1",
```

Comment:
for now yarn link react, react-dom, and stream-chat-react

Reply:
i see you resolved this but looks like they still not linked?

Reply:
also the TS version is old so can bump to latest and remove the carets on all em

Reply:
I'm tabling this for now. It's not compiling